### ISSUE TITLE
CI: "Ubuntu 16.04 LTS environment is deprecated and will be removed on September 20, 2021. Migrate to ubuntu-latest."

### ISSUE THREAD

**Issue by thac0** [oneapi-src/oneDNN#1149]
# Summary
Noted a spooky warning in your CI system. This link will vanish in 10 days apparently, but here's the link anyway (check the warnings tab in the summary):

https://dev.azure.com/mkldnn/dnnl/_build/results?buildId=670&view=results

The message in full for posterity:
```
##[warning]Ubuntu 16.04 LTS environment is deprecated and will be removed on September 20, 2021. Migrate to ubuntu-latest instead. For more details, see https://github.com/actions/virtual-environments/issues/3287.
```

# Version
N/A

# Environment
Your CI builders that use Ubuntu Xenial (16.04)

# Steps to reproduce

1. Submit a PR.
2. See observed warnings from the Xenial builders.

# Observed behavior
See above.

# Expected behavior
All builders ought to be fully supported by the provider. Might I recommend 20.04 builders be added and the 16.04 builders dropped?

**Comment by densamoilov** [comment]
Thanks @thac0 for letting us know. We have an issue created in the internal tracker. Will adjust or testing accordingly.

**Comment by vpirogov** [comment]
Fixed by b704033824d8f198070bfd04f07a71346c9c82bb.
